.gitignore

```diff
@@ -1,5 +1,5 @@
 # Build files
-/target
+**/target
```

Comment:
I believe just `target` should be fine. Or `target/` if you want to be directory specific.

Reply:
Ah, I was confused because I had some residual `examples/*/target` dirs back from when they used to be standalone.

Reply:
Yeah, considering we don't have those anymore, it's unclear if we should be specific and ignore just the root target _folder_ (`/target/`) or a target folder at any depth (`target/` or `**/target/`) or any file (dirs inclusive) named target (`target`).

Do you prefer the opt-out approach of starting strict and revising the specifier when we have an actual need for a file/folder named target, which is unlikely.

Or ignore specifically for our case and selectively revise the specifier to include more and more cases as we discover them.

I'm leaning towards the first. But to be folder specific – `target/`.